crates/astria-composer/src/searcher/executor.rs

```diff
@@ -164,28 +183,46 @@ impl Executor {
                             break Err(e).wrap_err("failed submitting bundle to sequencer");
                         }
                     }
+                    // reset timer for preempt bundle if it doesn't fill up within the sequencer block time
+                    block_timer.reset_immediately();
```

Comment:
The interval timer is IMO the wrong thing to use here. I think a countdown timer makes more sense.